### DISCUSSION TITLE
Developer mode needs to be enabled to debug on Windows

### DISCUSSION THREAD

**Discussion by MrFahrenheit-55** [serverpod/serverpod#698] [Q&A] [answered]
Hi, I am new to Serverpod and I am testing on Windows. I want to run and debug my application in VS Code without much success.

**Describe the bug**
What plugins are used on Serverpod for the need to have developer mode enabled?

**To Reproduce**
1. Run and debug the main.dart file in the *app*_flutter directory

**Expected behaviour**
Expect to run and debug the application

**Screenshots**
![image](https://user-images.githubusercontent.com/83400730/221975732-b369097f-1daf-456a-94ce-da426ee56905.png)

**Serverpod versions**
 -  Serverpod version: 1.0.1
 - serverpod_flutter: ^1.0.1

**Platform information**
Windows 11 Pro - server and application are running locally

**Comment by yahu1031** [comment, marked as answer]
This is not a serverpod issue. Looks like you are trying to run windows desktop application. Please enable developer settings in windows setting.
